server/lib/matchmaking/matchmaking-status.ts

```diff
@@ -45,49 +45,54 @@ export default class MatchmakingStatus {
     }
   }
 
+  async _getStatus(type: MatchmakingType) {
```

Comment:
You don't need to name these things this way any more, just make them actually `private`
```suggestion
  private async getStatus(type: MatchmakingType) {
```